### PR TITLE
fix(ProtoSerializer): serializers on models with properties not working

### PR DESCRIPTION
**Pull request by legau**

### CODE REVIEW

File: django_socio_grpc/proto_serializers.py [hunk 1]
```diff
@@ -76,59 +85,59 @@ def populate_dict_with_none_if_not_required(self, data_dict, message=None):
...
+            # INFO - AM - 11/03/2024 - Here we set the default value especially for the blank authorized data.
+            # We debated about raising a ValidaitonError but prefered this behavior.
+            # Can be changed if it create issue with users
+            return message.DESCRIPTOR.fields_by_name[field.field_name].default_value
+
+        elif field.allow_null or (field.default in [None, empty] and field.required is True):
+            if is_update_process or field.default not in [None, empty]:
+                return None
+            try:
+                deferred_attribute = getattr(self.Meta.model, field.field_name)
```
Comment (AMontagu) [inline django_socio_grpc/proto_serializers.py:134]: Meta and model in meta can be not set no ?

File: django_socio_grpc/tests/fakeapp/grpc/fakeapp.proto [hunk 2]
```diff
@@ -251,7 +251,7 @@ message CustomNameForResponse {
 // Test comment for whole message
 message CustomRetrieveResponseSpecialFieldsModelResponse {
     optional string uuid = 1;
-    optional int32 default_method_field = 2;
+    int32 default_method_field = 2;
```
Comment (AMontagu) [inline django_socio_grpc/tests/fakeapp/grpc/fakeapp.proto:254]: is this normal ?

File: django_socio_grpc/proto_serializers.py [hunk 3]
```diff
@@ -76,59 +85,59 @@ def populate_dict_with_none_if_not_required(self, data_dict, message=None):
...
-                    deferred_attribute = getattr(self.Meta.model, field.field_name)
-                    if deferred_attribute.field.default != NOT_PROVIDED:
-                        data_dict[field.field_name] = get_default_value(
-                            deferred_attribute.field.default
-                        )
-                        continue
-
-                data_dict[field.field_name] = None
-        return data_dict
+            with contextlib.suppress(_NoDictData):
```
Comment (AMontagu) [inline django_socio_grpc/proto_serializers.py:104]: can you add a comment on that line to explain what it does but mostly why do we need it this way  ?

File: django_socio_grpc/proto_serializers.py [hunk 4]
```diff
@@ -76,59 +85,59 @@ def populate_dict_with_none_if_not_required(self, data_dict, message=None):
             )
 
         is_update_process = (
-            hasattr(self.Meta, "model") and self.Meta.model._meta.pk.name in data_dict
+            hasattr(self.Meta, "model") and get_model_pk(self.Meta.model).name in data_dict
         )
+        clean_dict = {}
```
Comment (AMontagu) [inline django_socio_grpc/proto_serializers.py:90]: can you add a comment to explain why we need to instanciate a new object ?

File: django_socio_grpc/proto_serializers.py (hunk 1 above)
Comment (legau) [reply django_socio_grpc/proto_serializers.py:134]: that's why it's in a try except

File: django_socio_grpc/tests/fakeapp/grpc/fakeapp.proto (hunk 2 above)
Comment (legau) [reply django_socio_grpc/tests/fakeapp/grpc/fakeapp.proto:254]: yes, default_method_field is a serializer method field with a non optional return type

File: django_socio_grpc/proto_serializers.py (hunk 4 above)
Comment (legau) [reply django_socio_grpc/proto_serializers.py:90]: to keep the method functional

File: django_socio_grpc/proto_serializers.py (hunk 3 above)
Comment (legau) [reply django_socio_grpc/proto_serializers.py:104]: This is from ruff

**Pull request merged**